sdks/nodejs/src/agonesSDK.js

```diff
@@ -19,7 +19,26 @@ const services = require('../lib/sdk_grpc_pb');
 
 class AgonesSDK {
 	constructor() {
-		this.client = new services.SDKClient('localhost:59357', grpc.credentials.createInsecure());
+		function port() {
```

Comment:
I would consider moving this to a method to make it easier to test.
Either way we should add a test to maintain code coverage

Reply:
I've moved it to a separate method. I thought doing it inline was cute since it reduced scope, but maintaining test coverage is definitely more important.